README.rst

```diff
@@ -12,6 +12,27 @@ submit work log entries to Tick, via `Tick API`_.
 
 It is based on https://github.com/ProgrammersOfVilnius/gtimelog2jira.
 
+Installation
+============
+
+Prerequisities
+++++++++++++++
+
+.. code-block:: sh
+
+  brew install \
+  gobject-introspection \
+  gtk+3 \
+  libsoup \
+  libsecret \
```

Comment:
```suggestion
```

These dependencies are only needed for `gtimelog`, `gtimelog2tick` just needed `requests` which is declared in `setup.py`.

Reply:
True, let's find another place for that somewhere.